CapabilityAgent.js

```diff
@@ -203,7 +195,7 @@ async function _keyFromSeedAndName({seed, keyName, keyType}) {
   const signature = new Uint8Array(
     await crypto.subtle.sign(hmacKey.algorithm, hmacKey, nameBuffer));
   // generate Ed25519 key from HMAC signature
-  const keyPair = await cryptoLd.generate({seed: signature, type: keyType});
+  const keyPair = await Ed25519VerificationKey2020.generate({seed: signature});
```

Comment:
Are we sure we want to hardcode the suite here? What happens when next suite comes along, and we have to support both?

Reply:
Having key agility didn't even work properly here. When trying to load from the cache, the key type was not integrated in any way with the cache so if another incompatible mechanism were passed in, it would have broken. This also isn't about suite changes, it's purely about key algorithms. The suite doesn't matter, all that matters is that it's ed25519. Furthermore, it's not clear that anything other than ed25519 would work with this seed => key generation; it's a special property of ed25519 (or at the very least, all the APIs around it). So -- I'm going with YAGNI here as the simplest fix and path forward.

If ed25519 becomes a problem, we'll figure out what to do in the future, but there's no sensible replacement right now.

Reply:
Got it, makes sense!